src/components/MessageBox.js

```diff
@@ -69,31 +66,56 @@ class MessageBox extends React.Component {
 
     return _merge({}, {
       component: {
-        color: theme.Colors.WHITE,
-        boxSizing: 'border-box'
+        color: theme.Colors.GRAY_900,
+        boxShadow: theme.ShadowMed,
+        boxSizing: 'border-box',
+        borderTop: 'none',
+        borderRadius: 4,
+        display: 'flex',
+        flexDirection: 'column'
       },
-      header: {
+      alertbar: {
         background: color,
-        display: 'flex',
-        cursor: this.props.children ? 'pointer' : 'auto',
-        padding: theme.Spacing.XSMALL,
-        alignItems: 'center'
+        border: theme.Colors.GRAY_300,
+        color: theme.Colors.GRAY_900,
+        borderTop: '1px solid ' + color,
+        borderTopLeftRadius: 4,
+        borderTopRightRadius: 4,
+        height: 5
       },
-      leftHeader: {
-        flex: 1,
+      button: {
+        marginTop: this.props.isSmall ? 10 : null,
+        whiteSpace: 'nowrap',
+        alignSelf: this.props.isSmall ? null : 'flex-end',
+        textAlign: this.props.isSmall ? 'center' : null
+      },
+      header: {
         display: 'flex',
-        alignItems: 'center'
+        padding: theme.Spacing.SMALL,
+        paddingBottom: 0
       },
       title: {
-        fontFamily: theme.Fonts.SEMIBOLD,
+        fontFamily: theme.Fonts.SEMIBOLD
+      },
+      messageBody: {
+        alignItems: 'baseline',
+        display: 'flex',
+        flex: '1 1 auto'
+      },
+      messageContent: {
+        boxSizing: this.props.isSmall ? null : 'content-box',
+        textAlign: 'left',
+        width: this.props.isSmall ? '100%' : '75%',
+        paddingRight: this.props.isSmall ? 30 : null,
         fontSize: theme.FontSizes.MEDIUM
       },
-      icon: {
-        fill: theme.Colors.WHITE
```

Comment:
It looks like you removed this `icon` style but are still using it above for the `Icon` component.  Based upon your screen shots, it looks like the icons are defaulting to black which also points to this.  Can we add this back with the appropriate fill color?

Reply:
actually, we no longer want the icon fill to adjust to the color of the status anymore. we want them to remain solid Gray900 with a white icon, or solid white fill with Gray900 as the color

Reply:
But you're still applying `styles.icon` to the Icon component on line 37 but not providing the icon key here in the Styles object.  

This probably works internally because we're passing the icon key in the Styles prop object that we pass but here in the OS docs there is no icon style being applied so the fill of the icon becomes black, not `GRAY_900`.

Reply:
thanks for clarifying that for me. I updated it so that the icon has `fill: GRAY_900`. does that look better?

also, I noticed two my keys I accidentally left in the docs, so I removed those too